### PR TITLE
Migrating Results Table into its own file

### PR DESCRIPTION
**Pull request by dvargas92495**
Next Steps:
- [x] Sanity check Regressions, particularly the double footer

### CODE REVIEW

File: src/components/QueryDrawer.tsx [hunk 1]
```diff
@@ -255,9 +253,6 @@ const SavedQuery = ({
         hideResults={minimized}
         results={results.map(({ id, ...a }) => a)}
         preventExport
-        ctrlClick={(r) => {
-          clearOnClick?.(r.text);
-        }}
```
Comment (dvargas92495) [inline src/components/QueryDrawer.tsx:260]: replaced with `-action` selections

File: src/components/ResultsTable.tsx [hunk 2]
```diff
@@ -0,0 +1,526 @@
...
+  setPage: (p: number) => void;
+  onRefresh: () => void;
+  allResultsLength: number;
+  // @deprecated - move to `-action` columns through selections
+  extraColumn?: {
+    reserved: RegExp[];
+    width: number;
+    row: (e: Result) => React.ReactNode;
+    header: React.ReactNode;
+  };
```
Comment (dvargas92495) [inline src/components/ResultsTable.tsx:332]: I think over time, we need to decide which of these settings are actually for all layouts, and which ones are specific to table

File: src/components/ResultsView.tsx [hunk 3]
```diff
@@ -1173,41 +698,42 @@ const ResultsView: ResultsViewComponent = ({
...
-                      Showing {paginatedResults.length} of {results.length}{" "}
-                      results
-                    </i>
-                  </span>
-                </div>
-                {showContent && <QueryUsed parentUid={parentUid} />}
-              </div>
             </>
           ))}
+        <div style={{ background: "#eeeeee80" }}>
```
[the remark this answers is not in the document]
Comment (mdroidian) [reply src/components/ResultsView.tsx:712]: My suggestion would be to add 4px padding to bring it more in line with the rest.
`ResultsView`
`padding: 0 4px` on ln 716

And I think that was the padding for the ResultsTable footer.
`ResultsTable`
`padding: 8px 4px` on ln 452
`padding-left: 7px` on ln 453

image
image

**Review by mdroidian — CHANGES_REQUESTED** [review]
Requested changes in comments

**Pull request merged**